esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/metoffice.cylc

```diff
@@ -21,12 +25,17 @@
         [[[directives]]]
             --mem = 2G
```

Comment:
This `get_esmval` task probably only needs a minute or two at the MO; would that be worth changing here? Or did we decide to keep it consistent with the JASMIN directoves? 🤔

Reply:
This was an oversight. Time reduced in [b54b8e6](https://github.com/ESMValGroup/ESMValTool/pull/3562/commits/b54b8e63512ff86ea5f1f3b5cb3fb20b8447f9c3).